article.py

```diff
@@ -73,8 +74,8 @@
 
 class Article:
 
-    """ An Article represents a new article typically scraped from a web site,
-        as it is tokenized, parsed and stored in the Greynir database. """
+    """An Article represents a new article typically scraped from a web site,
+    as it is tokenized, parsed and stored in the Greynir database."""
```

Comment:
You might want to check your version of Black and update it - there was a bug in Black that caused comments to be misformatted like this.

Reply:
I was wondering about that. Thought that Black enforced a single, strict standard but then the rules apparently changed at some point. I now have black 21.9b0 installed via brew, hopefully that'll fix the issue.

Reply:
Still same behavior after upgrading to absolutely latest version. Is this the new standard formatting???

https://pypi.org/project/black/

Reply:
Are you adding any arguments to black in VSCode?